provider/flags/provider.go

```diff
@@ -31,33 +32,27 @@ func parseFlags(args []string) map[string]string {
 
 		key     string
 		inParam bool
-
-		pushSingleKey = func() {
-			res[key] = "true"
-			inParam = false
-		}
 	)
 
 	for _, arg := range args {
-		v, ok := parseArg(arg)
 
-		if ok {
-			if inParam {
```

Comment:
why did you delete those lines? (L45-47 & L57-60) it is important for key/value flags